.github/workflows/smoke.yml

```diff
@@ -41,291 +41,291 @@ jobs:
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'npm_and_yarn/**'
-          nuget:
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'nuget/**'
-          pip:
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'python/**'
-          'pip-compile':
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'python/**'
-          pipenv:
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'python/**'
-          pnpm:
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'npm_and_yarn/**'
-          poetry:
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'python/**'
-          pub:
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'pub/**'
-          submodules:
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'git_submodules/**'
-          swift:
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'swift/**'
-          terraform:
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'terraform/**'
-          'yarn':
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'npm_and_yarn/**'
-          'yarn-berry':
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'npm_and_yarn/**'
-          'yarn-berry-workspaces':
-            - .github/workflows/smoke.yml
-            - .dockerignore
-            - Dockerfile.updater-core
-            - 'common/**'
-            - 'updater/**'
-            - 'npm_and_yarn/**'
+      - uses: actions/checkout@v3
```

Comment:
I configured `yamllint` to not try to enforce a particular style for indentation, as long as the file was internally consistent.

So here for example, the problem is that the indentation varied depending on the section of the file.